_admin-center/analytics/module-codes.md

```diff
@@ -38,7 +38,6 @@ This page lists all the module codes and their full names.
 | QRTD  | Routed Queries |
 | SPEL  | Spelling Suggestions (Search.gov) |
 | SREL  | Related Searches |
-| TWEET | Tweets (Twitter) |
```

Comment:
@dawnpm Previously, we've kept the deprecated codes in the table (and marked with an asterisk), in order to interpret prior analytics that might have that module code.

Reply:
added back in, and added a note about why they're still there.